### DISCUSSION TITLE
Wish: RG300

### DISCUSSION THREAD

**Discussion by thesnable** [eduardofilo/RG350_adam_image#86] [Ideas & Requests]
Support for the RG300? Would be awesome

**Comment by eduardofilo** [comment]
That device it is supported in ODBeta, but it uses a different bootloader, so I would need to make a separate image, and I don't have that machine to test. I'm sorry.

**Comment by SuperBadger77** [comment]
I have a machine and can test it for you if it helps?

**Comment by eduardofilo** [comment]
Thanks, but it is too much uncontrolled. The different standalone emulators that I don't know about, the different RetroArch package that would have to be unpacked and adapted, SimpleMenu that I couldn't even run. Too many things to do blindly.

**Comment by thesnable** [comment]
contact me via email. I buy you one.

**Comment by eduardofilo** [comment]
Thanks, but it is not a matter of money, but time. And my holidays ends this week :-)
Thank you very much indeed.

**Comment by thesnable** [comment]
Ah a pity. But i tried. 😁

**Comment by SuperBadger77** [comment]
God loves a trier @thesnable. @eduardofilo I think you've earnt another holiday after this - great work and much appreciated :-)
